### ISSUE TITLE
No module - tomaty.lib.serialization

### ISSUE THREAD

**Issue by kramberg27** [ejmg/tomaty#53]
I installed after lots of trial and error. Just include a more extensive dependency list (e.g. gcc, alsa, python3-devel). I just didn't want to make a seperate issue.

_Not the main issue..._

When running, it can't find the library for serialization.

**Output**
```
Traceback (most recent call last):
  File "/usr/bin/tomaty", line 11, in <module>
    load_entry_point('tomaty==1.0.0', 'console_scripts', 'tomaty')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 565, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.6/site-packages/tomaty/tomaty.py", line 23, in <module>
    from tomaty.lib.serialization import tomaty_serialization
ModuleNotFoundError: No module named 'tomaty.lib.serialization'
```

**Comment by ejmg** [comment]
Hey Karl, thanks for reporting this.

I edited your comment for you, but anything that is a lot of lines of code/logs you should wrap around with three back-ticks for readability. It helps anyone who has to read it.

As for the error, I've patched it as of  bfd296f55b1c04b682eb74baa5f670128772b5f2. You can update to the proper version via `pip install --no-cache-dir tomaty==1.0.1` if `pip install --upgrade tomaty` does not work.

Finally, as for those other dependencies you needed: honestly put, those are _**very**_ basic dependencies that would be considered "redundant" to post. If you are doing any kind of real development on linux or mac, you will eventually need them. This just happens to be your first project that needed them! For future reference, whenever you need to do development with X tool/language, you will often need `x-dev` or `x-dlib` etcetera. The naming isn't constant, but you get the idea clearly from `python3-dev`. 

Thanks for reporting the bug, unfortunately local testing can only do so much. I hadn't bothered to install the app again via pypi myself after pushing the major release. It was luckily just a dumb mistake.

Actual question: Did you do this on a n*x system or Windows?

**Comment by kramberg27** [comment]
Thanks! This worked.

- Sorry about the formatting. I know basic markdown, just derped.
- I know lots of the dependencies are basic, but it's not clear that I needed **development** editions to just **install** tomaty.
- What would be really helpful is listing actual package names to install (e.g. `python3-cairo-devel`, `python-gobject`, or `alsa-lib-devel`).
- I did this on Linux (Solus).
